ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -309,7 +304,7 @@
 								}
 							]
 						},
-						"description": "Endpoint defined in draft-ietf-oauth-introspection-04, used to retrieve metadata about a token, such as approved scopes and the context in which the token was issued.\n\nGiven an access token, a client can perform an HTTP POST on /oauth2/introspect?token=access_token to retrieve a JSON object.\n"
+						"description": "Endpoint defined in RFC7662, used to retrieve metadata about a token, including:/n/n*Approved scopes./n/n* The user that authorized the token./n/n*Expiry time./n/n*Proof of possession JSON Web Key./*/*The context in which the token was issued.\n\nGiven an access token, a client ID and a client secret, you can retrieve a JSON object with a HTTP POST on /oauth2/introspect?token=access_token endpoint.\n"
```

Comment:
Since the user profile likely does not have detailed knowledge of OIDC or RFC7662, I'd suggest changing this to:

Used to retrieve metadata about an access token, including:/n/n*Approved scopes./n/n* The user that authorized the token./n/n*Expiry time./n/n*Proof of possession JSON Web Key./*/*The context in which the token was issued.\n\nGiven an access token, a client ID and a client secret, you can retrieve a JSON object with a HTTP POST on /oauth2/introspect?token=access_token endpoint.\

For detailed information regarding the specification behind this endpoint please refer to [RFC 7662 OAuth 2.0 Token Introspection ](https://tools.ietf.org/html/rfc7662) 

Also, please note the /n characters are not rending as newlines/carriage returns in postman.

Reply:
In addition the example output does not render the JSON in pretty print format.

Reply:
Ref -- this is for POST Introspect -- I simplified this to:

Returns the content of a user access token, including scopes. Requires the Client ID and Client Secret for an existing web app.

The example response is now "pretty print" formatted.